media-libs/gst_plugins_good/gst_plugins_good-1.22.0.recipe

```diff
@@ -22,18 +22,82 @@ libVersion="0.2200.0"
 
 PROVIDES="
 	gst_plugins_good$secondaryArchSuffix = $portVersion
+	lib:libgstadaptivedemux2$secondaryArchSuffix
+	lib:libgstalaw$secondaryArchSuffix
+	lib:libgstalpha$secondaryArchSuffix
+	lib:libgstalphacolor$secondaryArchSuffix
+	lib:libgstapetag$secondaryArchSuffix
+	lib:libgstaudiofx$secondaryArchSuffix
+	lib:libgstaudioparsers$secondaryArchSuffix
+	lib:libgstauparse$secondaryArchSuffix
+	lib:libgstautodetect$secondaryArchSuffix
+	lib:libgstavi$secondaryArchSuffix
+	lib:libgstcairo$secondaryArchSuffix
+	lib:libgstcutter$secondaryArchSuffix
+	lib:libgstdebug$secondaryArchSuffix
+	lib:libgstdeinterlace$secondaryArchSuffix
+	lib:libgstdtmf$secondaryArchSuffix
+	lib:libgsteffectv$secondaryArchSuffix
+	lib:libgstequalizer$secondaryArchSuffix
+	lib:libgstflac$secondaryArchSuffix
+	lib:libgstflv$secondaryArchSuffix
+	lib:libgstflxdec$secondaryArchSuffix
+	lib:libgstgdkpixbuf$secondaryArchSuffix
+	lib:libgstgoom$secondaryArchSuffix
+	lib:libgstgoom2k1$secondaryArchSuffix
+	lib:libgsthaikuaudiosink$secondaryArchSuffix
+	lib:libgsticydemux$secondaryArchSuffix
+	lib:libgstid3demux$secondaryArchSuffix
+	lib:libgstimagefreeze$secondaryArchSuffix
+	lib:libgstinterleave$secondaryArchSuffix
+	lib:libgstisomp4$secondaryArchSuffix
+	lib:libgstjpeg$secondaryArchSuffix
+	lib:libgstlame$secondaryArchSuffix
+	lib:libgstlevel$secondaryArchSuffix
+	lib:libgstmatroska$secondaryArchSuffix
+	lib:libgstmonoscope$secondaryArchSuffix
+	lib:libgstmpg123$secondaryArchSuffix
+	lib:libgstmulaw$secondaryArchSuffix
+	lib:libgstmultifile$secondaryArchSuffix
+	lib:libgstnavigationtest$secondaryArchSuffix
+	lib:libgstoss4$secondaryArchSuffix
+	lib:libgstpng$secondaryArchSuffix
+	lib:libgstreplaygain$secondaryArchSuffix
+	lib:libgstrtp$secondaryArchSuffix
+	lib:libgstshapewipe$secondaryArchSuffix
+	lib:libgstsmpte$secondaryArchSuffix
+	lib:libgstsoup$secondaryArchSuffix
+	lib:libgstspectrum$secondaryArchSuffix
+	lib:libgstspeex$secondaryArchSuffix
+	lib:libgsttaglib$secondaryArchSuffix
+	lib:libgsttwolame$secondaryArchSuffix
+	lib:libgstudp$secondaryArchSuffix
+	lib:libgstvideobox$secondaryArchSuffix
+	lib:libgstvideocrop$secondaryArchSuffix
+	lib:libgstvideofilter$secondaryArchSuffix
+	lib:libgstvideomixer$secondaryArchSuffix
+	lib:libgstvpx$secondaryArchSuffix
+	lib:libgstwavenc$secondaryArchSuffix
+	lib:libgstwavparse$secondaryArchSuffix
+	lib:libgstxingmux$secondaryArchSuffix
+	lib:libgsty4menc$secondaryArchSuffix
```

Comment:
All these libs aren't in $libDir, not sure why they have been added here?

Reply:
Sorry, I didn't know that libraries in subfolders weren't supposed to be included.

Reply:
These plugins are not really usable by anything but gstreamer. We use the "addon:" namespace for Haiku add-ons, could be used here. But I don't see the need, but nothing else depends on such plugins.